### PR TITLE
Cleanup plot types docs

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

- Do not show a partial signature in the title. Something like
  `contour[f]([X, Y], Z, [levels=], ...)` is rather confusing.
- Add a link to the docstring of the respective function.
- Minor text cleanups on the gallery page.

### CODE REVIEW

**Comment by jklymak** [comment]
The partial sig was from the cheatsheet, and an argument for it would be that it makes the PlotTypes page more useful as a quick reference without having to drill down a level.

**Comment by timhoffm** [comment]
The different media cheatsheet and website have different capabilities and use cases. Thus, we should consider to adapt the content and presentation accordingly.

The cheatsheet does not support links to defer to further information. (Whether or not desired) it has to provide all information at top level. The content and detail is chosen as an appropriate compromise between information and detail. It heavily uses styling to denote

grafik

In contrast, the gallery title cannot be styled and is wrapped. This makes it much less readable.

grafik

Even more, the additional information is only a help in very specific situations. But the additional clutter distracts from the core message, i.e. announcing and distinguishing the different plot functions by name. Compare

grafik

vs.

grafik

I argue that the additional indirection is worth the clarity and is actually not too bad since the user is already navigating in the browser.

You could alternatively make a single cheat-sheet like page with more top-level content. However, then sphinx-gallery is not the right tool and you'd have to build and layout that page from scratch in HTML.

**Comment by jklymak** [comment]
Sure I understand your point.  I just don't feel the Args are unclear as a quick reference.  But we can see what other folks think

**Comment by story645** [comment]
I'd prefer if minimally the data kwargs were left in since that's a major distinguishing factor between different plot types, but I also see Tim's point so won't block if it's left out.

**Comment by jklymak** [comment]
@anntzer suggest keeping the signature but dropping the square brackets...

File: plot_types/arrays/contourf.py [hunk 1]
```diff
@@ -1,7 +1,14 @@
 """
-=====================================
-contour[f]([X, Y], Z, [levels=], ...)
-=====================================
+========================<===========
```
Comment (jklymak) [inline plot_types/arrays/contourf.py:2]:
```suggestion
====================================
```

**Review by jklymak — APPROVED** [review]

**Pull request merged**